dbt/models/iasworld/schema/iasworld.pardat.yml

```diff
@@ -282,14 +282,18 @@ sources:
           - name: tieback
             description: '{{ doc("shared_column_tieback_key_pin") }}'
             tests:
-              - expression_is_true: 
+              - expression_is_true:
                   name: iasworld_pardat_tieback_does_not_contain_hyphen
                   expression: not like '%-%'
                   additional_select_columns: *select-columns
                   config: *unique-conditions
                   meta:
                     category: incorrect_values
                     description: tieback should not contain hyphens
+                  tags:
+                    # Valuations can't act on these failures since hyphens are
+                    # not distinguishable in the iasWorld UI
+                    - test_qc_exclude_from_workbook
```

Comment:
I'm thinking about emailing Mirella to ask if we can act on these ourselves, since there are <1k of them. Does that sound OK to you?

Reply:
Yes absolutely, let's suggest that we provide a mass upload with the current and then corrected versions.

Reply:
Done! I also wonder if perhaps we should update our consuming code to strip the hyphens -- as far as I can tell we don't do that yet.